types/vue3-openlayers/package.json

```diff
@@ -0,0 +1,8 @@
+{
+    "private": true,
+    "dependencies": {
+        "vue": "3.*",
+        "ol": "6.*",
```

Comment:
`ol` has its types on DefinitelyTyped, so it shouldn't be put in the `package.json`

```suggestion
```

Reply:
thanks! done